docs/images/setPriorityLevelSequenceDiagram.drawio

```diff
@@ -1 +1 @@
\ No newline at end of file
+<mxfile host="app.diagrams.net" modified="2020-04-08T09:36:29.765Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36" etag="IsuZUEHFnnHOOmfLaUHO" version="12.9.0" type="device"><diagram id="o_KrSBOa8JoWa7Z2Ccje" name="Page-1">3VxRc5s4EP41mck9NAMICXhM3Ka9mfQmN22n13vDINtcseXBcmL3159khAEJB+wi4SEPrVmEAO1+u9+uJG7AZLn7mIXrxWcS4/TGseLdDXh/4zg+guxfLtjnAtcHuWCeJXEuskvBl+QXFkJLSLdJjDe1hpSQlCbrujAiqxWOaE0WZhl5rTebkbR+13U4x4rgSxSmqvR7EtOFkNooKE98wsl8IW7tO15+YhkWjcWbbBZhTF4rIvDhBkwyQmj+a7mb4JSPXTEu+XWPJ84eHyzDK9rlgq8/4O57sN+s7eWvv9/9+3XyiUbvbFc8HN0Xb4xjNgDikGR0QeZkFaYfSulDRrarGPNuLXZUtnkiZM2ENhP+hyndC22GW0qYaEGXqTgbh5vF4Xp+gFfxPVcUO1yRFc4lj0maiv7V9xSvviHbLMJvvVzejr9Q5UIxOh8xWWKa7VmDDKchTV7qGg+F4cyP7Y6XPpOEPYpjCRuHwsSPFm7Ve6BhNsdUXFQqiP2oPEUpOqjtHBXmd3sJ0614g/uIkkzRK7O/Nf+5XaZ5A/DwgjOaMFt/Cqc4fSabhCZkxZpMCaVkyRqk/MRDGP2cH3Q+ISm/jvUGZoe/Sh/3aTLn11JuA1Vln9QfvxLv3tSMOOtatREuEPVaAhII0aICRWSd1mRNCWePuDdi0EDXDGqQU1eqadj4CmxuwP1zlpAsofsJWS7DVayouVQiV8LrIqH4yzo8jOcrC391hc2YOuqIcaKIyTc0Iz9x5UyMpgiifqACkYQVWwWL7TSgxdWGFn/MaAnMoMXzh0VL0ISWx5QNchRm8VOyof1ixY9wM1amPnThmzrrjhVPckHe4FAJRgwVxxBUnGBYqNhOE1a+Jb0CJIbYj90mgPjOFKCegokDWwHim8SHA0aMD9dQuuKCYfHhNOLjOcw2WE1afo9wYXSCcHnB1OopiADvyoKIA0cMEgeZAYntDQwStwkkn8Nk1W8YCbE/a4QIinw8nfUDERtcWRgZJCOpgCJixHmTRF8XTJ1VbOTN2Cv+w29zdzz8cTi0bFAI3u/Ec+RH++rRM84SNk446w1sua1fA7WTsqCjYbWgko17uK80W/MGm/Pv89i5vS0Zaf4EvbqIAh5XacUGjM4YX5KyRMfSY3Sn7nPK6Fra6zG6QjvVuCRbYcVaxDszMXy4ge+5pOKdSvOr+KyBQhSqF/GPnqVK4xpiFJA9UH9BasxF5oLetEeWofiePHlwAuStHdm21FE+NtqIIxiE/XeMCjXr+g3rCYa1nrPphDR/VdjISTrxdntNnr2hxKvZs5upYfmXeXZt2QcYhLeZmnJ3ukJzqGrwpY5d7si4Y1eZF3vuv/COPiUrfPuHYlQMH7RuA3WcCc1XQSlEoZjKj5j6cXWdQDHHv0zi+GCWTdWGuqn2gWD7xMhXEOw2INjRhmCv3VWqEVGOmCLNf8cTe6+e6fNFUn1m+hLIOsNTVUllyGHDkBey310bICkcdoOaSsaQ+3ZHmqt4QF1ksMH8+dZinQH7meIXvmJurOB1O0TfpvK4PvCqPGe25aPYvOJjNIqwZR4EVU0EJhXhDpLgluXVusuFrQ5Xc93L7eqafS3E6exqlTQ17LSUSFvaayqRqmsjFZvrN6cxM+foOnUsH0uSLTmNpw3LDQRoNDlNYTTXU5GWZ/A70prWjkznNC5Q8Lnm6wBEML6tBOYx5zdyucdvQHMTnPWFZpW3KitjUcpHf5qxX3P+63ZWXw04UZuw/8MlH8LVdMP/S9g47g4trXU6YgW7QKJeQYOCiza12QVdCoa2MtrXUyWuzID73jERzmlaELiXZMY9FZ7droVnY5FA3hkhR/ie5iaR1XwfvezNeEXazBJ9ZEvsrSETM+ruoaMMdDT2dFjJR4ZOh6HKhVQf3VJUPDkurX7ITF3P9lBt0EFHvqn6Ixu83ZHmwh4aJN/RXoOAXafljW17kRIUIM/K9RTf5J0gxX1O2V9Lez3xsNDO2OMhaNqy1uCUbH2+eMxLqDuT2MF2gl5azpA7Utiw5nIGVKdo8Q5HWzrq6Vkg09mm7NYsn+1jfnaX0DwHheLoR+VMmXvyg33lQM48j8msfWcdU1cxy+u0TzpozGU7TwV3XURlhkO6J1LRszmkkgHI9qiZQ0K1yDb6rEv5ksPQWZd3zUS+J6TDgQP+2WRcJv0tyyVb2msi4+aLU2b2xHsSGXe7kXFtpepiLdAouXhnZA71mYlLqbjckemZRaRW09hzHz9YcU//jHe3h/mnETNzJDEc0PDhJKPRFjXsHqp8RkTRxFlbW3sYMF9aeDj4Xm405k1ASM9KJsUXBRd+aaq1I9NOraieXyVZrSa6jpTp+sjRl+kaN59i68LZkxYOvCs2vpV93SEvKP9gJ8s8l1Err9CyWM+Db7bXw6hRwxfZembUPYSIQKp2uR2L1fpi6pi/GYW6bgYfLJZcSpAHjyXq0tgNe26xyuuJb0q4HfWiLE/aVQQaytZmS1E9LAAoqtYsBiO3UrdmIRiAy2vX1p0FnWpI55uWrIuq15f7gs6T0519gaHNTFIqDGT62ZU/BFLl27X8Tr6nN5cBFQOdnUzdRuMnlCRaX8maHZafA8+1Vn5THXz4Hw==</diagram></mxfile>
```

Comment:
Do you want to color the `Flashcard` class as well hahaha

Reply:
Coloured >_<